.github/workflows/release.yaml

```diff
@@ -0,0 +1,88 @@
+name: Release a new version
+
+on:
+  release:
+    types:
+      - published
+
+env:
+  NODE_OPTIONS: '--max-old-space-size=8192'
+
+jobs:
+  publish-to-npm:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check workflow runs in a release branch
+        run: |
+          if [[ "${{ github.event.release.target_commitish }}" != *releases* ]]; then
```

Comment:
This workflow requires a release branch.

IIRC, this means we'll need to create the tag by targeting a release branch rather than `main`.
Is there anything else that we'll need to do?

Reply:
you're correct, we would create a release branch and then we would create a new release via Github releases. It actually create a tag.

Reply:
Nice, it's not so different to the current flow.